test/reporters/console/no-props.js

```diff
@@ -1,7 +1,7 @@
 const consoleReporter = require('../../../src/reporters/console-reporter')
 const fixture = require('../../fixtures/reporters/console/empty')
 
-describe('Console Reporter', () => {
+describe('/fixtures/reporters/console/empty', () => {
```

Comment:
This is the name of the fixture, not the file containing the code. You can just modify it to `describe('/reporters/console')`